docs/modules/ROOT/pages/subscriptions/filtering.adoc

```diff
@@ -177,4 +187,525 @@ movieUpdate(where: {
+----
+movieRelationshipDeleted(where: { movie: { genre_IN: ["Comedy", "Adventure"] }, createdRelationship: { actors: { node: { name: "Jim Carrey" } } } }) {
+    movie {
+        title
+    }
+    deletedRelationship {
+        actors { 
+            screenTime
+            node { 
+                name
+            }
+        }
+    }
+}
+----
+
+NOTE: `Where` will only filter by existing properties right before deletion.
+
+
+=== Relationship-related filters
+So far we've seen how to filter the subscription events by using operations on nodes or relationship properties. The relationship-related filtering logic is even more powerful, as filters can also express the expected relationship field, or the expected concrete type at the other end of the relationship when having relationships to Abastract Types.
+
+The following examples are valid for both `CREATE_RELATIONSHIP`/ `DELETE_RELATIONSHIP` events. Their purpose is to illustrate the various ways in which a subscription to a relationship event can be filtered to suit needs of various specificity.
+
+Considering the following type definitions:
+[source, graphql, indent=0]
+----
+type Movie {
+    title: String
+    genre: String
+    actors: [Actor!]! @relationship(type: "ACTED_IN", properties: "ActedIn", direction: IN)
+    directors: [Director!]! @relationship(type: "DIRECTED", properties: "Directed", direction: IN)
+    reviewers: [Reviewer!]! @relationship(type: "REVIEWED", properties: "Review", direction: IN)
+}
+interface ActedIn @relationshipProperties {
+    screenTime: Int!
+}
+type Actor {
+    name: String
+}
+type Person implements Reviewer {
+    name: String
+    reputation: Int
+}
+union Director = Person | Actor
+interface Directed @relationshipProperties {
+    year: Int!
+}
+interface Reviewer {
+    reputation: Int!
+}
+type Magazine implements Reviewer {
+    title: String
+    reputation: Int!
+}
+interface Review {
+    score: Int!
+}
+----
+
+And the base subscription operation:
+
+[source, graphql, indent=0]
+----
+movieRelationshipDeleted(where: {}) {
+    movie {
+        title
+    }
+    deletedRelationship {
+        actors { 
+            screenTime
+            node { 
+                name
+            }
+        }
+        directors {
+            year
+            node {
+                ... on PersonEventPayload { # generated type
+                    name
+                    reputation
+                }
+                ... on ActorEventPayload { # generated type
+                    name
+                }
+            }
+        }
+        reviewers {
+            score
+            node {
+                reputation
+                ... on MagazineEventPayload { # generated type
+                    title
+                    reputation
+                }
+                ... on PersonEventPayload { # generated type
+                    name
+                    reputation
+                }
+            }
+        }
+    }
+}
```

Comment:
I'm not sure about the specific `Where` typename here, but I think this example works best if you say - "given the following subscription, you can use the following `where` inputs in the GraphQL variable values to get different results" - the current way that random JSON is used in the below examples is a little confusing.

You'll also need to indent the whole query over a bit.

```suggestion
subscription MovieRelationshipDeleted($where: MovieRelationshipDeletedWhere) {
movieRelationshipDeleted(where: {}) {
    movie {
        title
    }
    deletedRelationship {
        actors { 
            screenTime
            node { 
                name
            }
        }
        directors {
            year
            node {
                ... on PersonEventPayload { # generated type
                    name
                    reputation
                }
                ... on ActorEventPayload { # generated type
                    name
                }
            }
        }
        reviewers {
            score
            node {
                reputation
                ... on MagazineEventPayload { # generated type
                    title
                    reputation
                }
                ... on PersonEventPayload { # generated type
                    name
                    reputation
                }
            }
        }
    }
}
}
```